example/Makefile.inc

```diff
@@ -13,4 +13,8 @@ AM_CFLAGS = \
 	-I$(top_srcdir)/platform/@with_platform@/arch/@ARCH_DIR@ \
 	-I$(top_builddir)/include
 
+if CODE_INSTRUM
+AM_LDFLAGS = -L$(LIB) -lssl -lcrypto -latomic
```

Comment:
I suggest to pick up dynamic-examples-tests PR into your branch and then base this code on top.

Reply:
No need at this point (can be done at master ->caterpillar merge time). Most likely, #390 will get into Caterpillar before this PR.